apps/meteor/client/lib/VideoConfManager.ts

```diff
@@ -287,17 +292,21 @@ export const VideoConfManager = new (class VideoConfManager extends Emitter<Vide
 			newData[attributeName] = value;
 		}
 
-		debug && console.log(`[VideoConf] Updating attribute "${attributeName}" of call "${callId}".`);
+		this.debugLog(`[VideoConf] Updating attribute "${attributeName}" of call "${callId}".`);
 		this.incomingDirectCalls.set(callId, newData);
 	}
 
+	private emitError(error = 'error-videoconf-unexpected'): void {
+		this.emit('error', { error });
```

Comment:
Consider emitting a subclass of `RocketChatError` (from `apps/meteor/client/lib/errors`) instead.